Gh-408 Update home and intro pages (#409)
gchq/gaffer-doc: docs/administration-guide/introduction.md

# Administration Guide

> This guide is intended for more administrators of a Gaffer deployment and more

Reviewer: ```suggestion
This guide is aimed at administrators of a Gaffer deployment and more
```
Phrasing of this first sentence is a little weird and could do with a little rewrite :)
